robot/EDA/resources/EDA.robot

```diff
@@ -174,7 +174,7 @@ Get Affl Records Count
     ${NS} =                 Get EDA namespace prefix
     [Arguments]             ${field_name}
     ${result} =             SOQL Query
-    ...                     SELECT COUNT(${NS}${field_name}) FROM ${NS}Affl_Mappings__c
+    ...                     SELECT COUNT(${field_name}) FROM ${NS}Affl_Mappings__c
```

Comment:
I'm not sure about removing the NS variable from here. It seems it was working fine previously.

Reply:
just for reference the field_name in this case is name which doesnt require namespace